### DISCUSSION TITLE
How to make renovate discover all my python requirements files ?

### DISCUSSION THREAD

**Discussion by MathKlim** [renovatebot/renovate#18171] [Request Help] [answered]
I'm switching from dependabot to renovate and I'm struggling a bit to configure it.

Most of my python project have multiple requirements files, having always the same name structure, eg : 

* requirements.txt
* requirements-dev.txt
* requirements-doc.txt
* requirements-test.txt

Right now renovate only catch the first one, The doc says to add a `renovate.json` file at the root of the project, which I've done, with the following config.

```json
{
  "pip_requirements": {
    "fileMatch": ["(^|/)requirements(-.*)?.txt$"]
  }
}
```

This regex works on [regex101](https://regex101.com/r/27SgOb/1), but renovate still only catches requirements.txt. 

Did I miss something ?

**Comment by rarkins** [comment, marked as answer]
Your config works fine for me: https://github.com/renovate-reproductions/reqs/issues/3

Please create a reproduction

**Reply by MathKlim** [reply]
It's okay, I found the (dumb) mistake. This regex was not in the main branch, but in my dev branch, so renovate wouldn't catch it.

Thanks anyway.
